### PR TITLE
Add reference to react components' css classes

### PR DESCRIPTION
**Pull request by iHiD**
Add info about `c-` css classes

### CODE REVIEW

**Review by kntsoriano — APPROVED** [review]

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -98,8 +98,9 @@ View Components are either functional and written in React, or non-functional an
 
 ### React components
 
-- The JS for each component lives in `app/javascript/components/**/XXX.js`
-- The CSS for each component lives in `app/css/components/**/XXX.css`
+- The JS for each component lives in `app/javascript/components/**/XXX.js`.
+- The CSS for each component lives in `app/css/components/**/XXX.css`.
```
Comment (ErikSchierboom) [inline docs/CONTRIBUTING.md:102]: Is this a requirement for tailwind? Personally, I would suggest having the CSS for the component in the same directory as the component itself, as they're so tightly bound together.

**Review by ErikSchierboom — APPROVED** [review]

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (iHiD) [reply docs/CONTRIBUTING.md:102]: No, I just don't know how to do that.

If you can work out how to do that, I'd be :+1: on that too. I'll merge this and you can update it if you can fix it 🙂

Comment (ErikSchierboom) [reply docs/CONTRIBUTING.md:102]: Shall I create an issue?

Comment (iHiD) [reply docs/CONTRIBUTING.md:102]: Please :)

**Pull request merged**